### PR TITLE
[14.0] Add product_seasonality

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: product_seasonality/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Product seasonility",
```
Comment (jcoux) [inline product_seasonality/__manifest__.py]:
```suggestion
    "name": "Product seasonality",
```

File: product_seasonality/models/res_company.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    default_seasonal_config_id = fields.Many2one(
+        string="Default seasonal configuration", comodel_name="seasonal.config"
+    )
+
+    def _create_default_seasonal_conf(self):
+        self.ensure_one()
+        if not self.default_seasonal_config_id:
+            self.default_seasonal_config_id = self.env["seasonal.config"].create(
+                {"name": f"Default seasonal configuration: {self.name}"}
```
Comment (jcoux) [inline product_seasonality/models/res_company.py]: Maybe we want to translate this term?

File: product_seasonality/models/seasonal_config.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Camptocamp SA
+# @author: Julien Coux <julien.coux@camptocamp.com>
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class SeasonalConf(models.Model):
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config.py]:
```suggestion
class SeasonalConfig(models.Model):
```

File: product_seasonality/models/seasonal_config_line.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        self.ensure_one()
+
+        weekday = date.strftime("%A").lower()
+        # check if we are on the correct day of the week
+        if not self[weekday]:
+            return False
+
+        # check if the provided date is between date_start and date_end
+        # of the seasonal conf line
+        # TODO: include limits w/ `=`?
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py]: Seems good for me like this: if your seasonal products can be sold from the 2021-06-01, the 2021-05-31 must return False, but 2021-06-01 yes.

File: product_seasonality/models/seasonal_config_line.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    @api.constrains("date_start", "date_end")
+    def _check_date_end(self):
+        for line in self:
+            if line.date_end and line.date_end < line.date_start:
+                raise ValidationError(
+                    _("The end date cannot be earlier than the start date.")
+                )
+
+    def is_sale_ok(self, date):
+        # TODO: can't we just get a date and determin week day from it?
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py]: With the current way it's easy to check if the right boolean is checked or not: `self["monday"]` by example will directly give you the value of the flag.

**Review by jcoux — CHANGES_REQUESTED** [review]
About terms, can you rename each location of "Seasonal" by "Product seasonal".
And the translation can be "Produits de saison".
==> I will give you all the French translations after English renaming ;)
By example rename model "seasonal.config" by "product.seasonal.config", etc...

**Comment by simahawk** [comment]
@jcoux reworded and exported translations from scratch

File: product_seasonality/i18n/fr.po [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__date_start
+msgid "Date Start"
+msgstr ""
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_res_company__default_seasonal_config_id
+msgid "Default product seasonal configuration"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Configuration des produits de saison par défaut"
```

File: product_seasonality/i18n/fr.po [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+#. module: product_seasonality
+#: model:ir.model,name:product_seasonality.model_res_partner
+msgid "Contact"
+msgstr ""
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__create_uid
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__create_uid
+msgid "Created by"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Créé par"
```

File: product_seasonality/i18n/fr.po [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__create_date
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__create_date
+msgid "Created on"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Créé le"
```

File: product_seasonality/i18n/fr.po [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__date_end
+msgid "Date End"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Date de fin"
```

File: product_seasonality/i18n/fr.po [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__date_start
+msgid "Date Start"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Date de début"
```

File: product_seasonality/i18n/fr.po [hunk 11]
```diff
@@ -0,0 +1,213 @@
...
+#: model:ir.model.fields,field_description:product_seasonality.field_res_company__default_seasonal_config_id
+msgid "Default product seasonal configuration"
+msgstr ""
+
+#. module: product_seasonality
+#: code:addons/product_seasonality/models/res_company.py:0
+#, python-format
+msgid ""
+"Default product seasonal configuration is required: you can't remove it."
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "La configuration des produits de saison par défaut est requise : vous ne pouvez la supprimer."
```

File: product_seasonality/i18n/fr.po [hunk 12]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: code:addons/product_seasonality/models/res_company.py:0
+#, python-format
+msgid "Default seasonal configuration: %s"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgid "Default product seasonal configuration: %s"
msgstr "Configuration des produits de saison par défaut : %s"
```

File: product_seasonality/i18n/fr.po [hunk 13]
```diff
@@ -0,0 +1,213 @@
...
+#: model:ir.model.fields,field_description:product_seasonality.field_res_partner__display_name
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__display_name
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__display_name
+msgid "Display Name"
+msgstr "Nom affiché"
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__friday
+msgid "Friday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Vendredi"
```

File: product_seasonality/i18n/fr.po [hunk 14]
```diff
@@ -0,0 +1,213 @@
...
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config____last_update
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line____last_update
+msgid "Last Modified on"
+msgstr "Dernière modification le"
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__write_uid
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__write_uid
+msgid "Last Updated by"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Dernière modification par"
```

File: product_seasonality/i18n/fr.po [hunk 15]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__write_date
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__write_date
+msgid "Last Updated on"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Dernière mise à jour le"
```

File: product_seasonality/i18n/fr.po [hunk 16]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__line_ids
+msgid "Lines"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Lignes"
```

File: product_seasonality/i18n/fr.po [hunk 17]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__monday
+msgid "Monday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Lundi"
```

File: product_seasonality/i18n/fr.po [hunk 18]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config__name
+msgid "Name"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Nom"
```

File: product_seasonality/i18n/fr.po [hunk 19]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__product_id
+msgid "Product"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Produit"
```

File: product_seasonality/i18n/fr.po [hunk 20]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.seasonal_config_line_view_form
+msgid "Product Seasonal Line"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Ligne de produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 21]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.seasonal_config_line_view_tree
+msgid "Product Seasonal Lines"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Lignes de produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 22]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.view_partner_form
+msgid "Product seasonal availability is managed on"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "La disponibilité des produits de saison est gérée sur"
```

File: product_seasonality/i18n/fr.po [hunk 23]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.actions.act_window,name:product_seasonality.seasonal_config_line_act_window
+msgid "Product seasonal config line"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Ligne de configuration des produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 24]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.actions.act_window,name:product_seasonality.seasonal_config_act_window
+#: model:ir.model,name:product_seasonality.model_seasonal_config
+#: model:ir.model.fields,field_description:product_seasonality.field_res_partner__seasonal_config_id
+#: model:ir.model.fields,field_description:product_seasonality.field_res_users__seasonal_config_id
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__seasonal_config_id
+msgid "Product seasonal configuration"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Configuration des produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 25]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.seasonal_config_line_view_search
+msgid "Products"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Produits"
```

File: product_seasonality/i18n/fr.po [hunk 26]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__saturday
+msgid "Saturday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Samedi"
```

File: product_seasonality/i18n/fr.po [hunk 27]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.view_seasonal_config_view_form
+msgid "Seasonal"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgid "Product seasonal"
msgstr "Produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 28]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model,name:product_seasonality.model_seasonal_config_line
+msgid "Seasonal configuiration Line"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgid "Product seasonal configuration Line"
msgstr "Ligne de configuration des produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 29]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.seasonal_config_line_view_search
+#: model_terms:ir.ui.view,arch_db:product_seasonality.seasonal_config_view_tree
+msgid "Seasonals"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgid "Product seasonals"
msgstr "Produits de saison"
```

File: product_seasonality/i18n/fr.po [hunk 30]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__sunday
+msgid "Sunday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Dimanche"
```

File: product_seasonality/i18n/fr.po [hunk 31]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: code:addons/product_seasonality/models/seasonal_config_line.py:0
+#, python-format
+msgid "The end date cannot be earlier than the start date."
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "La date de fin ne peut pas être plus tôt que la date de début."
```

File: product_seasonality/i18n/fr.po [hunk 32]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__thursday
+msgid "Thursday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Jeudi"
```

File: product_seasonality/i18n/fr.po [hunk 33]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__tuesday
+msgid "Tuesday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Mardi"
```

File: product_seasonality/i18n/fr.po [hunk 34]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model:ir.model.fields,field_description:product_seasonality.field_seasonal_config_line__wednesday
+msgid "Wednesday"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "Mercredi"
```

File: product_seasonality/i18n/fr.po [hunk 35]
```diff
@@ -0,0 +1,213 @@
...
+
+#. module: product_seasonality
+#: model_terms:ir.ui.view,arch_db:product_seasonality.view_partner_form
+msgid "the parent company"
+msgstr ""
```
Comment (jcoux) [inline product_seasonality/i18n/fr.po]:
```suggestion
msgstr "la société parente"
```

File: product_seasonality/models/res_company.py [hunk 36]
```diff
@@ -0,0 +1,40 @@
...
+
+    default_seasonal_config_id = fields.Many2one(
+        string="Default product seasonal configuration", comodel_name="seasonal.config"
+    )
+
+    def _create_default_seasonal_conf(self):
+        self.ensure_one()
+        if not self.default_seasonal_config_id:
+            self.default_seasonal_config_id = self.env["seasonal.config"].create(
+                {"name": _("Default seasonal configuration: %s") % self.name}
```
Comment (jcoux) [inline product_seasonality/models/res_company.py]:
```suggestion
                {"name": _("Default product seasonal configuration: %s") % self.name}
```

File: product_seasonality/models/seasonal_config.py (hunk 3 above)
Comment (jcoux) [inline product_seasonality/models/seasonal_config.py]:
```suggestion
class SeasonalConfig(models.Model):
```

File: product_seasonality/models/seasonal_config_line.py [hunk 37]
```diff
@@ -0,0 +1,66 @@
...
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SeasonalConfLine(models.Model):
+    _name = "seasonal.config.line"
+    _description = "Seasonal configuiration Line"
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py]:
```suggestion
    _description = "Product seasonal configuration Line"
```

File: product_seasonality/models/seasonal_config_line.py [hunk 38]
```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 Camptocamp SA
+# @author: Julien Coux <julien.coux@camptocamp.com>
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SeasonalConfLine(models.Model):
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py]:
```suggestion
class SeasonalConfigLine(models.Model):
```

File: product_seasonality/models/seasonal_config_line.py [hunk 39]
```diff
@@ -0,0 +1,66 @@
...
+        self.ensure_one()
+
+        weekday = date.strftime("%A").lower()
+        # check if we are on the correct day of the week
+        if not self[weekday]:
+            return False
+
+        # check if the provided date is between date_start and date_end
+        # of the seasonal conf line
+        # TODO: include limits w/ `=`?
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py]: Seems good for me like this: if your seasonal products can be sold from the 2021-06-01, the 2021-05-31 must return False, but 2021-06-01 yes.
```suggestion
```

File: product_seasonality/tests/test_company.py [hunk 40]
```diff
@@ -0,0 +1,31 @@
...
+
+from .common import CommonCase
+
+
+class TestCompanyCase(CommonCase):
+    def test_create_default(self):
+        rec = self.env["res.company"].create({"name": "ACME Inc."})
+        self.assertEqual(
+            rec.default_seasonal_config_id.name,
+            "Default seasonal configuration: ACME Inc.",
```
Comment (jcoux) [inline product_seasonality/tests/test_company.py]:
```suggestion
            "Default product seasonal configuration: ACME Inc.",
```

File: product_seasonality/views/seasonal_config.xml [hunk 41]
```diff
@@ -0,0 +1,53 @@
...
+    # Copyright 2021 Camptocamp SA
+    # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+    <record id="seasonal_config_view_tree" model="ir.ui.view">
+        <field name="name">seasonal.config.tree</field>
+        <field name="model">seasonal.config</field>
+        <field name="arch" type="xml">
+            <tree string="Seasonals">
```
Comment (jcoux) [inline product_seasonality/views/seasonal_config.xml]:
```suggestion
            <tree string="Product seasonals">
```

File: product_seasonality/views/seasonal_config.xml [hunk 42]
```diff
@@ -0,0 +1,53 @@
...
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_seasonal_config_view_form" model="ir.ui.view">
+        <field name="name">seasonal.config.form</field>
+        <field name="model">seasonal.config</field>
+        <field name="arch" type="xml">
+            <form string="Seasonal">
```
Comment (jcoux) [inline product_seasonality/views/seasonal_config.xml]:
```suggestion
            <form string="Product seasonal">
```

File: product_seasonality/views/seasonal_config_line.xml [hunk 43]
```diff
@@ -0,0 +1,83 @@
...
+    <record id="seasonal_config_line_view_search" model="ir.ui.view">
+        <field name="name">seasonal.config.line.search</field>
+        <field name="model">seasonal.config.line</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="seasonal_config_id" />
+                <field name="product_id" />
+                <filter
+                    name="group_seasonal_config_id"
+                    string="Seasonals"
```
Comment (jcoux) [inline product_seasonality/views/seasonal_config_line.xml]:
```suggestion
                    string="Product seasonals"
```

**Review by jcoux — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
@jcoux changes included, good for you?

File: product_seasonality/models/seasonal_config_line.py [hunk 44]
```diff
@@ -0,0 +1,66 @@
...
+        if date < self.date_start or self.date_end and date > self.date_end:
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py]: Seems good for me like this: if your seasonal products can be sold from the 2021-06-01, the 2021-05-31 must return False, but 2021-06-01 yes.
```suggestion
        if date < self.date_start or self.date_end and date > self.date_end:
```

File: product_seasonality/models/seasonal_config_line.py [hunk 45]
```diff
@@ -0,0 +1,66 @@
...
+            return False
+
+        return True
+
+    # TODO: shall we use the product's company to retrieve the default config?
```
Comment (jcoux) [inline product_seasonality/models/seasonal_config_line.py:53]: Don't understand this point?

Comment (simahawk) [reply product_seasonality/models/seasonal_config_line.py:53]: A product is normally attached to a company. Here we could determine the config from the company.
It's just a TODO question for the future, not a blocking point now.

**Review by jcoux — APPROVED** [review]
Thecnically seems good for me: thanks for changes! :+1:

File: product_seasonality/readme/CONFIGURE.rst [hunk 46]
```diff
@@ -0,0 +1 @@
+TODO
```
Comment (damdam-s) [inline product_seasonality/readme/CONFIGURE.rst]: still to do?

**Review by damdam-s — APPROVED** [review]

File: product_seasonality/readme/CONFIGURE.rst (hunk 46 above)
Comment (simahawk) [reply product_seasonality/readme/CONFIGURE.rst]: done, thank you

**Pull request merged**